### PR TITLE
Update doc how_to_run_rector_in_docker

### PR DESCRIPTION
**Pull request by Ioni14**
Hello,

After some tries to run Rector, I finally did it with some changes in the command.

Here is what I have when I execute the proposed one : `docker run -it --rm -u $(id -u):$(id -g) -v $(pwd):/project rector/rector process src --config rector.php`
image

I've added workdir to rector folder and explicitly written the full path of project files.

I hope this can help some users.

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Hi, thank you for the PR!

I've added @JanMikes to review and decide, as he's the Docker author here :)

**Comment by JanMikes** [comment]
Hi, sorry im not going to merge this - as described in https://github.com/rectorphp/rector/issues/5003#issuecomment-752733167 this is workaround that is not reliable and might not 100% work for everyone.

What you do in this MR is replacing default usage with changing working directory everywhere.

To merge this PR, please change it this way - adding some title (just like `## Permission issues`), describe what issue docker users might run into (some example error message) and with info what they can try to fix (just like you did already) and that it might not be 100% that it will fix the issue.

File: docs/how_to_run_rector_in_docker.md [hunk 1]
```diff
@@ -6,20 +6,21 @@ To make sure you are running latest version, use `docker pull rector/rector`.
 *Note that Rector inside Docker expects your application in `/project` directory - it is mounted via volume from the current directory (`$pwd`).*
 
 ```bash
-docker run --rm -v $(pwd):/project rector/rector:latest process src --dry-run
+docker run --rm -v $(pwd):/project -w /rector rector/rector:latest process /project/src --dry-run
```
Comment (JanMikes) [inline docs/how_to_run_rector_in_docker.md:9]: Default and recommended usage should stay unchanged

**Comment by Ioni14** [comment]
Hi @JanMikes and thanks for your review.
I proposed this PR because the recommended usage throws me the error that I shared in the screenshot. Do you have an idea why ?

**Comment by mpdude** [comment]
@Ioni Could you give #5599 a try?

Check out the Rector source code on that branch.

Then, run `docker build --target rector-scoped -t rector-scoped-php8.0 .` to build a local Docker image named `rector-scoped-php8.0`.

Does that image work for you?

**Comment by JanMikes** [comment]
Hi, im closing this as it will be solved via #5599 - there will be scoped version in docker so you will not face this problem with autoloading priorities anymore once it got merged.

**Pull request closed without merging**